Bug in backend

@claude fix it

```
________ test_send_request_notifications_blocked_users_no_notification _________
db = None
    def test_send_request_notifications_blocked_users_no_notification(db):
        """
        Regression test: send_request_notifications should not send notifications
        when the host and surfer are not visible to each other (e.g., one blocked the other).
        """
        user1, token1 = generate_user()
        user2, token2 = generate_user()
    
        today_plus_2 = (today() + timedelta(days=2)).isoformat()
        today_plus_3 = (today() + timedelta(days=3)).isoformat()
    
        # Create a host request
        with requests_session(token1) as requests:
>           host_request_id = requests.CreateHostRequest(
                requests_pb2.CreateHostRequestReq(
                    host_user_id=user2.id, from_date=today_plus_2, to_date=today_plus_3, text="Test request"
                )
            ).host_request_id
src/tests/test_bg_jobs.py:1249: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
src/tests/test_fixtures.py:796: in fake_handler
    response = handler.unary_unary(request, context, session)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
src/couchers/servicers/requests.py:215: in CreateHostRequest
    context.abort_with_error_code(
src/couchers/context.py:142: in abort_with_error_code
    context.abort(status_code, error_message)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = <tests.test_fixtures.MockGrpcContext object at 0x7cd9f5be81d0>
code = <StatusCode.INVALID_ARGUMENT: (3, 'invalid argument')>
details = 'Host request cannot be shorter than 250 characters.'
    def abort(self, code, details):
>       raise FakeRpcError(code, details)
E       tests.test_fixtures.FakeRpcError: (<StatusCode.INVALID_ARGUMENT: (3, 'invalid argument')>, 'Host request cannot be shorter than 250 characters.')
src/tests/test_fixtures.py:738: FakeRpcError
```

--- a/app/backend/src/tests/test_bg_jobs.py
+++ b/app/backend/src/tests/test_bg_jobs.py
@@ -1248,7 +1248,7 @@ def test_send_request_notifications_blocked_users_no_notification(db):
     with requests_session(token1) as requests:
         host_request_id = requests.CreateHostRequest(
             requests_pb2.CreateHostRequestReq(
-                host_user_id=user2.id, from_date=today_plus_2, to_date=today_plus_3, text="Test request"
+                host_user_id=user2.id, from_date=today_plus_2, to_date=today_plus_3, text=valid_request_text()
             )
         ).host_request_id
 

